services/venice-router/src/main/java/com/linkedin/venice/router/api/VeniceDelegateMode.java

```diff
@@ -284,7 +286,7 @@ public <H, P extends ResourcePath<K>, K, R> Scatter<H, P, K> scatter(
           readRequestThrottler.mayThrottleRead(
               storeName,
               keyCount * readRequestThrottler.getReadCapacity(),
-              Optional.of(veniceInstance.getNodeId()));
+              perRouterStoreThrottlerEnabled ? null : veniceInstance.getNodeId());
```

Comment:
Does it matter to pass the NodeId all the time?
The implementation can decide to use it or ignore it, right?

Reply:
I think `ReadRequestThrottler` should deal with this config: `perRouterStoreThrottlerEnabled`, and it should be transparent to the throttler user: `VeniceDelegateMode`.